File: my js/pattern-problems/pattern-6.js
// const prompt = require("prompt-sync")();

let n = parseInt(prompt("Give me a number: "));

for (let row = 1; row <= n; row++) {
  for (let col = 1; col <= row; col++) {
    document.write("*");
  }
  document.write("<br/>");
}

// expected output
// if n= 4
// *
// * *
// * * *
// * * * *

File: my c++/pattern-problems/pattern-6.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    int row = 1;
    while (row<=n)
    {
        int col = 1;
        while (col<=row)
    {
        cout << "*"
             << " ";
        col++;
    }
    cout << endl;
    row++;
    }
}

// expected output
// if n= 4
// * 
// * * 
// * * *
// * * * *